lib/get-unified-config.js

```diff
@@ -422,14 +422,25 @@ function cleanseServiceName (serviceName) {
   return serviceName.toLowerCase().replace(/ /g, '-').replace(/[^a-z0-9.:_-]/g, '')
 }
 
-// mask the service key so it's not revealed when logging. do not presume a
-// a valid service key, i.e., 64-hex-digits:1-to-255-valid-name
+// mask the service key so it's not revealed when logging.
+// DO NOT presume a valid service key, i.e., 64-hex-digits:1-to-255-valid-name
```

Comment:
This comment is a bit outdated since in SWO the API Token is typically longer (but I'd also heard talk of generating much shorter ones).  It's probably safest to just assume that the service key needs to have a least 1 char, followed by `:`, followed by at least another char.